source/docs/building-and-previewing.md

```diff
@@ -39,6 +39,18 @@ Using the default site structure, `/build_local` will look like this:
     <div class="ellipsis">...</div>
 </div>
 
+### Watching files for changes
+
+To compile the assets and keep watching your project for changes you can run the following command:
+
+```bash
+npm run watch
+```
+
+Your project will open at a localhost URL and any time any file changes in your project, Webpack will recompile your assets, and Jigsaw will regenerate your static HTML pages to `/build_local`.
+
+You can also watch a specific environment by running `npm run local`, `npm run staging`, or `npm run prod`.
+
```

Comment:
```suggestion
```

Reply:
This isn't correct; I don't believe these don't run `watch`, and `npm run local` doesn't exist in the default build.

Reply:
Yeah this was already in the docs elsewhere 🤦🏻 thanks for trying to make it consistent @zuzana-kunckova, unfortunately the other docs were wrong though lol 🙃

Reply:
This is also in the Compiling Assets docs page. Since it doesn't work in any build, I'll remove it from there too.